src/pages/dev/send-tokens/interchain-tokens/roadmap.mdx

```diff
@@ -0,0 +1,22 @@
+# ITS Roadmap
+
+## Background
+
+Prior to the release of the Axelar [Interchain Token Service](https://github.com/axelarnetwork/interchain-token-service) (ITS), Axelar allowed for only the transfer of gateway tokens between chains. These gateway tokens are manually registered in Axelar’s consensus the list of these gateway tokens can be found [here](https://docs.axelar.dev/resources/contract-addresses/mainnet#assets). With the release of ITS, now any team can provide multichain functionality to their token, whether its deploying a fresh new token on multiple chains or wiring up a pre deployed token to ITS to provide bridging functionality for the token. Unlike gateway tokens, ITS tokens are not minted on the Axelar chain itself rather they are sent between chains via Axelar [General Message Passing (GMP)](https://docs.axelar.dev/dev/general-message-passing/overview). 
+
+
+## ITS Today
+
+With the release of ITS each registered chain has an instance of the [ITS contract](https://github.com/axelarnetwork/interchain-token-service) deployed on it. This ITS contract registers tokens across multiple chains via a unique `interchainTokenId`. TokenManagers which are deployed. Each chain your token is registered to will have its own [tokenManager](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/token-manager/TokenManager.sol) for that chain. The token manager handles multichain operations such as minting/burning the tokens and locking the tokens (in the event of a lock/unlock bridging pattern). 
```

Comment:
> TokenManagers which are deployed
Is missing statement / verb?
tokenManager -> `TokenManager` (capitalization and backticks)